dandiapi/api/tests/test_version.py

```diff
@@ -109,10 +111,16 @@ def test_version_rest_update(api_client, user, draft_version):
 
     new_name = 'A unique and special name!'
     new_metadata = {'foo': 'bar', 'num': 123, 'list': ['a', 'b', 'c']}
+    year = datetime.now().year
+    url = f'https://identifiers.org/{draft_version.dandiset.identifier}/draft'
     saved_metadata = {
         **new_metadata,
         'name': new_name,
         'identifier': f'DANDI:{draft_version.dandiset.identifier}',
+        'id': f'{draft_version.dandiset.identifier}/draft',
+        'version': 'draft',
+        'url': url,
+        'citation': f'{new_name} ({year}). Online: {url}',
```

Comment:
I could see an argument for just inserting the values from the existing metadata here, since (except for citation, due to year) you're really just testing that the values haven't changed. However, I don't feel strongly about this, just an observation.

Reply:
They are inserted on line 188? `**new_metadata`?